viadot/flows/prefect_logs.py

```diff
@@ -0,0 +1,136 @@
+from typing import Literal
+import prefect
+import pandas as pd
+import datetime
+
+from prefect import Flow, task, Task
+from prefect.utilities import logging
+
+from viadot.tasks.azure_data_lake import AzureDataLakeUpload
+
+from viadot.task_utils import add_ingestion_metadata_task, df_to_parquet
```

Comment:
If you want to split imports for the groups maybe the better idea is to split by `prefect imports`, `viadot imports`, `other`.
Here there are 4 blank lines that we can reduce.